README.md

```diff
@@ -30,3 +30,21 @@ See [the wiki](https://github.com/GoogleChrome/chrome-types/wiki) for more.
 Running the code requires Node 16+ as well as a working version of Python (3 is preferred, but 2.7+ should work) installed on your system.
 This has only been tested on Linux and macOS.
 Python is used to convert Chromium's internal IDL format to JSON.
+
+**Steps**
+
+1. Add library as a dependency.
+   ```bash
+   npm i chrome-types --save-dev
```

Comment:
```suggestion
   npm install --save-dev chrome-types
```

It's useful to use/suggest [canonical](https://docs.npmjs.com/cli/v9/commands/npm-install) command forms in documentation